Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.23 as backend-builder
+FROM golang:latest as backend-builder
```

Comment:
This is safe with modern go versions because we specify the toolchain in the go.mod (so it will build using the correct Go toolchian regardless of the actual version in the base container).